accepted/pt-BR/PSR-3-logger-interface.md

```diff
@@ -0,0 +1,299 @@
+Interface de Log
+===================
+
+Este documento descreve uma interface comum para bibliotecas de logging.
+
+O principal objetivo deste documento é permitir que bibliotecas recebam um objeto
+`Psr\Log\LoggerInterface` e escrevam logs para ele de uma maneira simples
+e universal. Frameworks e CMSs que têm necessidades específicas PODEM
+estender a interface para os seus próprios fins, mas DEVEM permanecer
+compatíveis com este documento. Isto garante que as bibliotecas de terceiros
+utilizadas possam escrever logs na aplicação principal.
+
+As palavras chave "DEVE", "NÃO DEVE", "OBRIGATÓRIO", "TEM QUE", "NÃO TEM QUE", "DEVERIA", "NÃO DEVERIA", "RECOMENDADO", "PODE" e "OPCIONAL" existentes neste documento devem ser interpretadas como são descritas na [RFC 2119][].
+
+A palavra `implementador` deste documento DEVE ser interpretada como
+uma implementação de `LoggerInterface` numa biblioteca ou framework.
+Usuários de loggers serão referidos como `user`.
```

Comment:
I'm in favor of _usuário_ on the entire document, but keeping the **code** in english, always.